src/app/editor/preview/layer-settings/settings-preview/settings-preview.component.ts

```diff
@@ -63,15 +63,17 @@ import { Store } from '@ngxs/store';
   ]
 })
 export class SettingsPreviewComponent implements OnInit {
-  currentLayer: SketchMSLayer;
-  data: SketchMSData;
-  isMouseOver: boolean;
+  previews: string[];
 
-  constructor(private store: Store) { }
+  constructor(private store: Store) {}
 
   ngOnInit() {
-    this.store.select(UiState.currentFile).subscribe(currentFile => {
-      this.data = currentFile;
+    this.store.select(UiState.currentData).subscribe(currentData => {
```

Comment:
could you explain why we introduced these changes?

Reply:
To keep consistency with the rest of the application. `currentFile` was used to hold the SketchMSData structure.